src/PKSim.Infrastructure/Serialization/Xml/Serializers/SimulationXmlSerializer.cs

```diff
@@ -57,7 +57,9 @@ public class IndividualSimulationXmlSerializer : SimulationXmlSerializer<Individ
       public override void PerformMapping()
       {
          base.PerformMapping();
-         MapEnumerable(x => x.AllCompoundPK, x => x.AddCompoundPK);
+         Map(x => x.AucIV);
```

Comment:
This would require project conversions
@Yuri05 I am ok to not do this at this stage. PEople would just have to rerun for individual simulations